module/dice/dice.mjs

```diff
@@ -207,7 +207,8 @@ export async function damageRoll({
 
   // Evaluate the configured roll
   for ( const roll of rolls ) {
-    await roll.evaluate({async: true});
+    const rollMode = roll.options.rollMode ?? defaultRollMode;
```

Comment:
```suggestion
    const rollMode = rolls.at(-1).options.rollMode ?? defaultRollMode;
```

Should be the same logic as in `DamageRoll.toMessage`. But I think both are equivalent, because all rolls should have the same roll mode here anyway.